website/docs/r/media_transform.html.markdown

```diff
@@ -0,0 +1,173 @@
+  description                 = "My transform description"
+  output {
+    relative_priority = "Normal"
+    on_error_type     = "ContinueJob"
+    preset {
+      type        = "BuiltInStandardEncoderPreset"
+      preset_name = "AACGoodQualityAudio"
+    }
+  }
+}
+
+```
+
+## Example Usage with Multiple Outputs
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "media-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestoracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_media_services_account" "example" {
+  name                = "examplemediaacc"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id         = azurerm_storage_account.example.id
+    is_primary = true
+  }
+}
+
+resource "azurerm_media_transform" "example" {
+  name                        = "transform1"
+  resource_group_name         = azurerm_resource_group.example.name
+  media_services_account_name = azurerm_media_services_account.example.name
+  description                 = "My transform description"
+  output {
+    relative_priority = "Normal"
+    on_error_type     = "ContinueJob"
+    preset {
+      type        = "BuiltInStandardEncoderPreset"
+      preset_name = "AACGoodQualityAudio"
+    }
+  }
+
+  output {
+    relative_priority = "Low"
+    on_error_type     = "ContinueJob"
+    preset {
+      type                = "AudioAnalyzerPreset"
+      audio_language      = "en-US"
+      audio_analysis_mode = "Basic"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `media_services_account_name` - (Required) The Media Services account name. Changing this forces a new Transform to be created.
+
+* `name` - (Required) The name which should be used for this Transform. Changing this forces a new Transform to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Transform should exist. Changing this forces a new Transform to be created.
+
+---
+
+* `description` - (Optional) An optional verbose description of the Transform.
+
+* `output` - (Required) One or more `output` blocks as defined below. At least one `output` must be defined.
+
+---
+
+A `output` block supports the following:
+
+* `on_error_type` - (Optional) A Transform can define more than one outputs. This property defines what the service should do when one output fails - either continue to produce other outputs, or, stop the other outputs. The overall Job state will not reflect failures of outputs that are specified with `ContinueJob`. Possibles value are `StopProcessingJob` or `ContinueJob`.
```

Comment:
I think this would be better ux and make more sense as
```suggestion
* `on_error_action` - (Optional) A Transform can define more than one outputs. This property defines what the service should do when one output fails - either continue to produce other outputs, or, stop the other outputs. The overall Job state will not reflect failures of outputs that are specified with `ContinueJob`. Possibles value are `StopProcessingJob` or `ContinueJob`.
```